resources/ts/Pages/Integrations/Detail.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { ReactNode, useState, useEffect } from "react";
+import Layout from "../../Components/Layout";
+import { Page } from "../../Components/Page";
+import { Heading } from "../../Components/Heading";
+import { faTrash } from "@fortawesome/free-solid-svg-icons";
+import { Integration } from "./Index";
+import { BasicInfo } from "../../Components/Integrations/Detail/BasicInfo";
+import { ContactInfo } from "../../Components/Integrations/Detail/ContactInfo";
+import { BillingInfo } from "../../Components/Integrations/Detail/BillingInfo";
+import { IntegrationInfo } from "../../Components/Integrations/Detail/IntegrationInfo";
+import { IntegrationSettings } from "../../Components/Integrations/Detail/IntegrationSettings";
+import { ButtonSecondary } from "../../Components/ButtonSecondary";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { useTranslation } from "react-i18next";
+
+type Props = { integration: Integration };
+
+const Detail = ({ integration }: Props) => {
+  const { t } = useTranslation();
+
+  const [isMobile, setIsMobile] = useState(false);
+
+  const handleResize = () => {
+    setIsMobile(window.innerWidth < 768);
+  };
+
+  useEffect(() => {
+    window.addEventListener("resize", handleResize);
+  });
```

Comment:
2 remarks:

1. Make sure to always include a dependency array. Otherwise the code inside will continuously run.
2. return a cleanup function to remove the event listener when the component unmounts

```suggestion
  useEffect(() => {
    window.addEventListener("resize", handleResize);
    return () => window.removeEventListener("resize", handleResize);
  }, []);
```